src/resources/Sources/Sources.ts

```diff
@@ -1,10 +1,145 @@
 import API from '../../APICore';
+import {New, PageModel} from '../BaseInterfaces';
 import Resource from '../Resource';
+import {ScheduleModel} from '../SecurityCache';
+import {
+    ActivityOperation,
+    CreateSourceOptions,
+    LightSourceModel,
+    ListSourcesParams,
+    RawSourceConfig,
+    SourceModel,
+} from './SourcesInterfaces';
 
 export default class Sources extends Resource {
     static baseUrl = `/rest/organizations/${API.orgPlaceholder}/sources`;
 
+    list(params?: ListSourcesParams) {
+        return this.api.get<SourceModel[]>(this.buildPath(Sources.baseUrl, params));

```

Comment:
This one shouldn't be implemented IMO it doesn't return the total entries and total pages so it doesn't make any sense.

Reply:
I'll remove it!